src/api/memberInstitutionsApi.ts

```diff
@@ -0,0 +1,25 @@
+import Axios from 'axios';
+import { getIdToken } from './userApi';
+import { StatusCode } from '../utils/constants';
+
+export enum MemberInstituionApiPaths {
+  MEMBER_INSTITUTION = '/member-institutions',
+}
+
+export const getAllMemberInstitutions = async () => {
```

Comment:
kanskje vi kan kalle det `getAllCustomerInstitutions` isteden? Synes member er så vagt

Reply:
works for me